### PR TITLE
added AbsoluteETA to widgets

### PR DESCRIPTION
**Pull request by tnestmeyer**
added an absolute ETA to the widgets (estimates the time when it will finish)

### CODE REVIEW

File: progressbar/widgets.py [hunk 1]
```diff
@@ -170,6 +170,29 @@ def __call__(self, progress, data):
...
+class AbsoluteETA(Timer):
+    '''Widget which attempts to estimate the absolute time of arrival.'''
+
+    def _eta(self, progress, data, value, elapsed):
+        """Update the widget to show the ETA or total time when finished."""
+        now = datetime.datetime.now()
+        if value == progress.min_value:
+            return 'Estimated finish time: ----/--/-- --:--:--'
+        elif progress.end_time:
+            return 'Finished at: %s' % self._format(datetime.datetime.now())
```
Comment (WoLpH) [inline progressbar/widgets.py]: Shouldn't that be `progress.end_time` instead of `datetime.now`?

**Comment by tnestmeyer** [comment]
Indeed, you're right WoLpH. Should give the same output, but your suggestion is cleaner. Directly fixed it!

**Comment by tnestmeyer** [comment]
added blank line before next class.

**Comment by WoLpH** [comment]
Merged and will release it soon :)

Thanks for the help!

**Comment by WoLpH** [comment]
It's now on Pypi. Sorry for the slow response, fixed Python 3 support as well ;)

**Pull request merged**